railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow `bin/rails test` to take absolute paths as arguments
```

Comment:
Should this have a changelog entry?  This fixes a regression, so it's not new behaviour since the last release.